core/dbt/cli/main.py

```diff
@@ -48,7 +48,7 @@ def __init__(
         self.manifest = manifest
         self.callbacks = callbacks
 
-    def invoke(self, args: List[str], **kwargs) -> Tuple[Optional[List], bool]:
+    def invoke(self, args: List[str], **kwargs) -> Tuple[Optional[Any], bool]:
```

Comment:
@stu-k updated this in https://github.com/dbt-labs/dbt-core/pull/7242 (rightly so!) so it now returns a `dbtRunnerResult` class. I think we want to update [this union](https://github.com/dbt-labs/dbt-core/blob/7a4d3bd2dc770885b74322b1c6811e80207ad342/core/dbt/cli/main.py#L52-L59) to now also include `Manifest`.

Reply:
Ahhh gotcha.  I thought you wanted it to be `Any` because it was going to be something different, but you didn't know what it was going to be and you didn't want things to break when Stu merged the other PR.  Makes more sense now.